sherpa/astro/ui/tests/test_astro_ui.py

```diff
@@ -257,20 +241,17 @@ def test_psf_model2d(self):
     # bug #12503
     def test_psf_pars_are_frozen(self):
         ui.load_psf('psf', ui.beta2d.p1)
-        self.assertEqual([], p1.thawedpars)
+        self.assertEqual([], ui.beta2d.p1.thawedpars)
```

Comment:
I understand your point, but that feature should have its own unit tests. I find it annoying that many files show up with "undefined global name"s in my IDE, even though they are fine. I'll change it to use get_model_component for now (not sure why I didn't in the first place), and I am sure there is plenty of places were the tests assume the pollution of the global namespace.